CareKitStore/CareKitStore/Structs/OCKOutcomeValue.swift

```diff
@@ -65,10 +65,16 @@ public struct OCKOutcomeValue: Codable, Equatable, CustomStringConvertible {
 
     /// The units for this measurement.
     public var units: String?
-
-    /// The date that this value was created.
+    
+    // The date that this value was created.
     public var createdDate = Date()
 
+    /// The sample’s start date.
```

Comment:
Can you change “sample” to “value” this documentation is related to OCKOutcomeValue

Reply:
Changed